pypasta/MailCharacteristics.py

```diff
@@ -373,11 +383,13 @@ def load_characteristics(config, clustering, message_ids = None):
     config.mbox_timewindow, and loads multiple instances of maintainers for the
     patches of the clustering.
     """
+    from .JailhouseMailCharacteristics import JailhouseMailCharacteristics
     from .LinuxMailCharacteristics import LinuxMailCharacteristics
     from .QemuMailCharacteristics import QemuMailCharacteristics
     from .UBootMailCharacteristics import UBootMailCharacteristics
     from .XenMailCharacteristics import XenMailCharacteristics
     _load_characteristics = {
+        'jailhouse': (load_maintainers_characteristics, JailhouseMailCharacteristics),
```

Comment:
And if we once have factored out those things, we might even see a better way to handle this dictionary here.